### PR TITLE
Increase the Set Ras Oob Config retries

### PR DESCRIPTION
**Pull request by abinayaddhandapani**
Increasing the SetRasOobConfig retries to 15 minutes as the 2P systems take more time to enable PCIE AER

### CODE REVIEW

File: src/runtime_errors.cpp [hunk 1]
```diff
@@ -73,13 +73,13 @@ oob_status_t SetOobConfig()
             sd_journal_print(LOG_ERR, "Failed to set ras oob configuration for "
                                       "Processor P0. Retrying....\n");
         }
-        sleep(10);
+        sleep(INDEX_20);
```
Comment (mahkurap) [inline src/runtime_errors.cpp:76]: I am not sure why INDEX_20 is used here? INDEX_20 is offset in the PPR record for a different purpose.  I dont think we should reuse that macro here.  May be we can define a new macro for 20 seconds and use it, it improves the readability of the program. 

struct error_section_descriptor
{
    uint32_t SectionOffset;
    uint32_t SectionLength;
    uint8_t RevisionMinor;
    uint8_t RevisionMajor;
    uint8_t SecValidMask;
    uint8_t Reserved;
    uint32_t SectionFlags;
    GUID_T SectionType;
    uint64_t FRUId[INDEX_2];
    uint32_t Severity;
    char FRUText[INDEX_20];
} __attribute__((packed));

File: src/runtime_errors.cpp [hunk 2]
```diff
@@ -58,7 +58,7 @@ oob_status_t SetOobConfig()
         oob_config.pcie_err_reporting_en = ENABLE_BIT;
     }
 
-    uint16_t retryCount = 30;
+    uint16_t retryCount = 45;
```
Comment (mahkurap) [inline src/runtime_errors.cpp:61]: how about increasing only the retryCount and keeping the sleep to 10 seconds.  That way, we are polling 10 more times, we will be quicker to respond back.  Also aren't we changing the behavior for single processor system by sleeping for longer time?

Comment (abinayaddhandapani) [reply src/runtime_errors.cpp:61]: Hi Mahesh. We are sending SetRasOobConfig command every 20 second until the command succeeds. Once the command is successful, the module starts the RAS functionality. For 1P system it takes around 180 seconds (9 retries for every 20 seconds ) and 600 seconds for 2P system (Around 30 retries for every 20 seconds) for the command to be successful. As it takes anywhere from 3 to 15 minutes for the PCIE AER to be enabled , I have used retries for every 20 seconds.

Comment (rajaganeshr) [reply src/runtime_errors.cpp:61]: Please test this once on dense platforms as well. init might be a factor of number of cores in soc.

Comment (abinayaddhandapani) [reply src/runtime_errors.cpp:61]: As per discussion with Mahesh, sleep of 20 seconds will not have any functional impact for both 1P and 2P system. So retaining the code changes of sleep 20.

**Review by mahkurap — APPROVED** [review]

File: src/runtime_errors.cpp [hunk 3]
```diff
@@ -73,13 +73,13 @@ oob_status_t SetOobConfig()
...
         }
-        sleep(10);
+        sleep(SLEEP_20);
         retryCount--;
     }
 
     if (num_of_proc == TWO_SOCKET)
     {
-        retryCount = 30;
+        retryCount = 45;
```
Comment (supven01) [inline src/runtime_errors.cpp:82]: nit: magic number 45.

**Review by supven01 — CHANGES_REQUESTED** [review]

File: src/runtime_errors.cpp (hunk 2 above)
Comment (abinayaddhandapani) [reply src/runtime_errors.cpp]: @rajaganeshr It is verified on dense platform as well

**Review by supven01 — APPROVED** [review]

**Pull request merged**